cogs/assistance.py

```diff
@@ -844,13 +844,20 @@ async def dump(self, ctx, console=None):
             embed.description = ("How to dump/build NSPs using NXDumpTool\n"
                                  "BAN Warning: only for use using offline emummc")
             await ctx.send(embed=embed)
-        elif self.check_console(console, ctx.channel.name, ('wiiu',)):
+        elif self.check_console(console, ctx.channel.name, ('wiiu')):
```

Comment:
You shouldn't have removed the comma. This will stop the command from automatically selecting wiiu while using the command in #wiiu-assistance

Reply:
Whoops! I'm sorry about that, I think I had made a few changes but then reverted them and got rid of the comma in the process. Hope I fixed it with my recent commit!